src/Workspaces/Core/Portable/Classification/IRemoteSemanticClassificationCacheService.cs

```diff
@@ -26,7 +26,7 @@ internal interface IRemoteSemanticClassificationCacheService
         /// <param name="checksum">Pass in <see cref="DocumentStateChecksums.Text"/>.  This will ensure that the cached
         /// classifications are only returned if they match the content the file currently has.</param>
         ValueTask<SerializableClassifiedSpans?> GetCachedSemanticClassificationsAsync(
-            SerializableDocumentKey documentKey,
+            DocumentKey documentKey,
```

Comment:
we had a dual of a primitive data type and it's 'serializable' equivalent.  however, the primitive data type was trivial to make serializable itself.  so this dual just went away.